docs/en/1_exist_data_model.md

```diff
@@ -317,6 +317,8 @@ Optional arguments:
 - `EVAL_METRICS`: Items to be evaluated on the results. Allowed values depend on the dataset, e.g., `proposal_fast`, `proposal`, `bbox`, `segm` are available for COCO, `mAP`, `recall` for PASCAL VOC. Cityscapes could be evaluated by `cityscapes` as well as all COCO metrics.
 - `--show`: If specified, detection results will be plotted on the images and shown in a new window. It is only applicable to single GPU testing and used for debugging and visualization. Please make sure that GUI is available in your environment. Otherwise, you may encounter an error like `cannot connect to X server`.
 - `--show-dir`: If specified, detection results will be plotted on the images and saved to the specified directory. It is only applicable to single GPU testing and used for debugging and visualization. You do NOT need a GUI available in your environment for using this option.
+- `--box-only`: If specified, plot only `bbox` on the images if the model outputs both `bbox` and `mask`. It is invalid to specify both `--box-only` and `--mask-only`.
```

Comment:
--box-only is a bit vague, would it be more precise to use `--show-box-only` and `--show-mask-only`?

Reply:
updated